metatron-app/metatron-discovery
# Fn fix add explore detail column sort

### Description

Add Sort function in metadata explore detail - column 
**Related Issue** : 

METATRON-2970

### How Has This Been Tested?

1. Explore Data 
2.Click random data card
3. change tab to column
4. check sorting function is working
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
